Source/WebCore/PAL/pal/text/TextCodecCJK.cpp

```diff
@@ -677,7 +675,8 @@ static Vector<uint8_t> shiftJISEncode(StringView string, Function<void(char32_t,
             continue;
         }
 
-        ASSERT(range.first + 3 >= range.second);
+        ASSERT(range.second - range.first <= 3);
```

Comment:
Do we also need to assert that `range.second >= range.first` to avoid underflow? Maybe not. I'm unclear on the types involved.